.circleci/config.yml

```diff
@@ -160,6 +171,24 @@ jobs:
     environment:
       NODE_ENV: staging
 
+  release_uat:
+    docker:
+      - image: cimg/node:18.16.0
+    resource_class: xlarge
+    steps:
+      - git_checkout_from_cache
+      - npm_install_from_cache
+      - build
+      - versioning:
+          version_name: uat
+      - persist_to_workspace:
+          root: packages
+          paths:
+            - core
+      - notify_slack
+    environment:
+      NODE_ENV: uat
```

Comment:
nitpick: not sure if this is needed, there is no condition that we need this node env

Reply:
I was following convention but I'll remove for now.